tests-policy

```diff
@@ -249,7 +249,8 @@ def update_known_issue(api, number, err, details, context, timestamp=None):
                         # only keep the last 10
                         if len(occurrences) > 10:
                             occurrences.pop(0)
-                        parts[part_idx] = f"{latest[0]}{latest_occurrences}{'\n'.join(occurrences)}"
```

Comment:
This change looks weird...

Reply:
Yeah, I'd also prefer to just build the whole `"\n".join()` as a separate value, but it felt too much like bikeshedding. We have lots of things in flight today..

Reply:
I would be happy to do that honestly. Was a bit suprised it was merged :)

Reply:
Oh.

```
>>> f'{"\n".join(["a", "b"])}'
'a\nb'
```

I see.....

Reply:
So this came in relatively recently, in a0223ff82431603b6926569ea8d8165b47e5566a.  And indeed, the syntax is new in Python 3.12.

Reply:
See https://github.com/cockpit-project/bots/pull/6269